webpack/assets/javascripts/react_app/components/AutoComplete/__tests__/__snapshots__/AutoCompleteReducer.test.js.snap

```diff
@@ -16,7 +14,7 @@ Object {
 
 exports[`AutoComplete reducer should handle AUTO_COMPLETE_INITIAL_QUERY 1`] = `
 Object {
-  "controller": "models",
+  "controller": null,
```

Comment:
is that supposed to be `null`?

Reply:
Some tests have failed after the refactor, this one should be deleted :)